src/table/__integ__/resizable-columns.test.ts

```diff
@@ -178,11 +192,25 @@ describe.each([true, false])('StickyHeader=%s', sticky => {
   );
 
   test(
-    'should render "width: auto" for the last on big screens and explicit value on small',
+    'sticky and real column headers must have identical widths on big screen',
+    setupStickyTest(async page => {
+      const stickyHeaders = await page.getFirstTableHeaders();
+      const realHeaders = await page.getLastTableHeaders();
+
+      expect(stickyHeaders.length).toBe(realHeaders.length);
+      expect(stickyHeaders.map(el => el.offsetWidth)).toEqual(realHeaders.map(el => el.offsetWidth));
```

Comment:
This test does not work as you expect. It compares `undefined` to `undefined`.

Why this happens – I will leave a learning opportunity for you

Reply:
Makes perfect sense - great catch!